openlibrary/templates/account/borrow.html

```diff
@@ -37,6 +37,10 @@
 th.status {
     width: 150px;
 }
+.LoanReadForm {
+    margin-top: 10px;
+    margin-bottom: 10px;
```

Comment:
Consider `margin: 10px 0;`

Reply:
I have shifted the above styles above to /static/css/legacy-borrowTable-adminUser.less scoped to #borrowTable like below:

```
body#admin,
body#user {
  #borrowTable {
    margin: 30px 0 25px;

    .LoanReadForm {
        margin: 10px 0;
    }
    /*.....rest of the styles from borrow.html as it is*/
    table {
    /*.....rest of the remaining styles pre-existing inside legacy-borrowTable-adminUser.less*/
 
```
, since it is already imported inside the static/css/page-user.less. 
Let me know if this seems fine.

Reply:
The BorrowTable as far as I understand it is captured in https://github.com/internetarchive/openlibrary/wiki/Design-Pattern-Library#borrowtable
The right component would therefore be:
static/css/components/borrowTable.less

(which links to this page)

Looking forward to your next patches!